test/unit/utils/utils.js

```diff
@@ -8,8 +8,8 @@ function isFieldDuplicated(field) {
   };
 }
 
-function loadAllFilesIn(location, featureFiles) {
-  return function loadFeatureFiles() {
+function loadAllFiles(location) {
+  return function loadFeatureFiles(featureFiles) {
```

Comment:
Here **location** parameter would be referenced when all the feature files listed are loaded. **featureFiles** would be a list of files other than the main file in the definition folder.

Reply:
Why do you need a function that return a function here?